repositories/noteRepository.js

```diff
@@ -0,0 +1,39 @@
+const Note = require('../models/Note');
+
+exports.createNote = async (title, content) => {
+  const note = new Note({ title, content });
+  return await note.save();
+};
+
+exports.getAllNotes = async (page, limit) => {
+    const skip = (page - 1) * limit; 
+    return await Note.find()
+      .skip(skip)         
+      .limit(Number(limit)); 
```

Comment:
It would also be better to sort the returned data, as it’s a good practice when implementing pagination.

Reply:
Done